.github/workflows/sec-check.yaml

```diff
@@ -0,0 +1,16 @@
+name: Security-Check
+# Manually triggered workflow - Treeverse members should only invoke it after checking for malicious code changes in other workflows.
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I don't understand.  This workflow runs on _any_ pull request, not just manually.

I *think* it would work if you added an _additional_ step that tests for the existence of a secret.  Then it would fail for a fork but work for manual triggering, no?